builtin/v8/market/policy.go

```diff
@@ -0,0 +1,69 @@
+package market
+
+import (
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/go-state-types/builtin"
+)
+
+// The number of epochs between payment and other state processing for deals.
+const DealUpdatesInterval = builtin.EpochsInDay // PARAM_SPEC
```

Comment:
the usual problem.

Reply:
These aren't really state, though I see why you wanted to export them similarly. Again, long term, we probably want the actors to encapsulate their logic/policy better, and access it via view methods.